src/scripts/meetups.js

```diff
@@ -0,0 +1,20 @@
+import { getCollection } from "astro:content";
+
+let allMeetups = await getCollection("meetup");
+allMeetups = allMeetups.map((meetup) => meetup.data)
```

Comment:
One idea is to use `meetup.slug` as an ID for the headlines at https://engineeringkiosk.dev/meetup/alps/archive/

This would make it possible to have direct links like https://engineeringkiosk.dev/meetup/alps/archive/#meetup4

Just an idea.